File: 프로그래머스/unrated/181926. 수 조작하기 1/수 조작하기 1.java
class Solution {
    public int solution(int n, String control) {
     for (char answer : control.toCharArray()) {
            switch (answer) {
                case 'w':
                    n = n + 1;
                    break;
                case 's':
                    n = n - 1;
                    break;
                case 'd':
                    n = n + 10;
                    break;
                case 'a':
                    n = n - 10;
                    break;
            }
        }
        return n;
    }
}

File: 프로그래머스/0/181926. 수 조작하기 1/수 조작하기 1.js
function solution(n, control) {
    for(var i = 0; i < control.length; i++)
    switch(control[i]){
        case "w":
            n++;
            break;
        case "s":
            n--;
            break;
        case "d":
            n+=10;
            break;
        case "a":
            n-=10;
            break;
        default:
            break;
    }
    return n;
}